applications/nrf_desktop/doc/config_channel.rst

```diff
@@ -78,167 +75,188 @@ The data is exchanged in HID feature reports. Each request has the following for
 | Report ID | Recipient | Event ID | Status | Data length | Data    |
 +-----------+-----------+----------+--------+-------------+---------+
 
-* Report ID - HID report ID of the feature report used for transmitting data.
-* Recipient - product ID of the device to which the request is addressed. Needed to route requests in a multi-device setup.
-* Event ID - identifier of the value that should be set or fetched - consists of a module ID and an option ID.
-* Status - used to exchange status of the request, also used by sender to ask for fetch.
-* Data length - indicates how many bytes of data the request holds.
-* Data - arbitrary length data connected to the request.
+Each feature report contains the following components:
+
+* Report ID - HID report identifier of the feature report used for transmitting data.
+* Recipient - Product identifier of the device to which the request is addressed.
+  Needed to route requests in a multi-device setup.
+* Event ID - Identifier of the value that should be set or fetched; consists of a module ID and an option ID.
+* Status - Value used to exchange status of the request; also used by sender to ask for fetch.
+* Data length - Value that indicates how many bytes of data the request holds.
+* Data - Arbitrary length data connected to the request.
 
-.. tip::
+.. note::
    Bluetooth Low Energy HID Service removes the leading report ID byte, resulting in firmware obtaining a data frame 1 byte shorter.
 
-   USB HID class transmits the whole report, including the report ID byte.
+   The USB HID class transmits the whole report, including the report ID byte.
+
 
 Handling configuration channel in firmware
 ==========================================
 
-To enable the configuration channel in nRF Desktop firmware, set the ``CONFIG_DESKTOP_CONFIG_CHANNEL`` Kconfig option.
-This option enables the :ref:`info` too.
+To enable the configuration channel in the nRF Desktop firmware, set the :option:`CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE` Kconfig option.
```

Comment:
CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE is the option I found in the code, but it still returns an error. Can you check, @MarekPieta ?

Reply:
Clarified via chat: nRF Desktop Kconfig options cannot be linked like the others.